build/OutputDirectories.props

```diff
@@ -25,5 +25,15 @@
       <TestPackagesDir>$(TestOutputDir)/packages/</TestPackagesDir>
       <TestArtifactsDir>$(TestOutputDir)/artifacts/</TestArtifactsDir>
       <TestResultXmlDir>$(TestOutputDir)/results/</TestResultXmlDir>
+      <ArchiveOutputDirectory>$(PackagesDirectory)</ArchiveOutputDirectory>
```

Comment:
I mean moving them out of &lt;Target&gt; (not necessarily the .targets file). Moving forward I want to do what you suggested, only move it to a global file if it's used in multiple places. But I'm sure I missed some of those (with previous PRs that have already merged). So I'll also take a separate pass to clean that up.